projects/plugins/jetpack/extensions/plugins/publicize/store/effects.js

```diff
@@ -65,24 +65,34 @@ export async function refreshConnectionTestResults() {
 		}
 
 		// Update post metadata.
-		dispatch( editorStore ).editPost( { jetpack_publicize_connections: connections } );
-
-		// Update connections in the piblicize store.
-		return dispatch( 'jetpack/publicize' ).setConnectionTestResults( connections );
+		return dispatch( editorStore ).editPost( { jetpack_publicize_connections: connections } );
 	} catch ( error ) {
-		// Refreshing connections failed
+		// console.error( error );
```

Comment:
probably, I'm not sure what we should do. Just in case and FYI, we haven't handled of that case from the beginning when we started to work on Publicize.

Reply:
I've removed this controversial change in order to move on 😅